src/bp/core/server.ts

```diff
@@ -140,6 +141,18 @@ export default class HTTPServer {
 
     this.app.use(monitoringMiddleware)
 
+    if (config.session.enabled) {
+      this.app.use(
+        session({
+          secret: botpressConfig.appSecret,
```

Comment:
We should use process.JWT_SECRET (or process.APP_SECRET), because the appSecret could be jwtSecret on the config file